dev-games/openscenegraph-openmw/openscenegraph-openmw-3.6_p20210610.ebuild

```diff
@@ -0,0 +1,162 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+LUA_COMPAT=( lua5-{1,2} )
```

Comment:
lua5-2 is dead, so you can remove it like with sol2. No need to update old ebuilds but on a bump it's nice to be up to date.

I do wonder if 3 or 4 works, but CMakeLists.txt does indicate it wants 5-2 or 5-1, so it may not be safe.